File: Exercicios_Em_Python_CV/desafio_011.py
#Faça um programa que leia a largura e altura de uma parede em metros
#calcule a sua área e quantidade de tinta necessária para pintá-la
#sabendo que cada litro de tinta pinta uma área de 2m²
largura = float(input("Digite a largura da parede em m:"))
altura = float(input("Digite a altura da parede em m:"))
area = largura * altura 
litros_de_tinta =area/2
print("Com a largura de {} m e a altura de {} m, temos uma área de {}m²".format(largura, altura, area))
print("logo precisamos de {}l de tinta".format(litros_de_tinta))

File: Exercicios_Em_Java_CV/Desafio011.java
package fundamentos;

import java.util.Scanner;

/*
 * Faça um programa que leia a largura e a altura de uma parede em
 * metros, calcule a sua área e a quantidade de tinta necessária para pintá-la
 * sabendo que cada litro de tinta, pinta uma área de 2m²
 */
public class Desafio011 {
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		System.out.print("Digite a largura:");
		float largura = scanner.nextFloat();
		scanner.nextLine();
		System.out.println("Digite a altura:");
		float altura = scanner.nextFloat();
		scanner.nextLine();
		float area = largura * altura;
		float tinta = area/2;
		System.out.println("area total :" + area + "m²");
		System.out.println("Total de tinta :" + tinta + "litros");
		scanner.close();
	}

}
